### PR TITLE
UnusedFunctionParametersCheck: report precise line

### PR DESCRIPTION
**Pull request by janedbal**

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
Since this is a BC break for people ignoring these errors, we have to do it in bleeding edge.

**Comment by janedbal** [comment]
> Since this is a BC break for people ignoring these errors, we have to do it in bleeding edge.

Done.

File: conf/config.neon [hunk 1]
```diff
@@ -1043,6 +1043,8 @@ services:
 
 	-
 		class: PHPStan\Rules\UnusedFunctionParametersCheck
+		arguments:
+			reportExactLine: %featureToggles.bleedingEdge%
```
Comment (ondrejmirtes) [inline conf/config.neon]: Please introduce a new toggle for that.

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: conf/config.neon (hunk 1 above)
Comment (janedbal) [reply conf/config.neon]: `reportPreciseLineForUnusedFunctionParameter` is there

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**